### PR TITLE
Start for PostgreSQL support

### PR DESCRIPTION
**Pull request by knutsel**
- All tests run clean
- CRConfig.json has no diffs
- Quick check in UI seems to look good. 

See README in traffic_control/app/pgdb for additional notes. 

@dew: see one commit with your name in it, i think the code is wrong for resolver4/6 (should be RESOLVER4/6)

### CODE REVIEW

**Comment by PSUdaemon** [comment]
This looks good from my perspective.

File: traffic_ops/app/pgdb/README.md [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+      
+      "target_description" : [
+          "Connection string to your PostgreSql database",
+          "Please ensure, that you have defined your connection string properly.",
+          "Ensure, that details like options='[double dash]client_encoding=UTF8' are included in your connection string (if necessary)."
+      ],
+      "target" : "pgsql:host=localhost;port=5432;dbname=to_development;options=--client_encoding=UTF8,postgres,your_postgres_passwd",
+      
+      "encoding_description" : [
+          "PHP encoding type.",
```
Comment (dewrich) [inline traffic_ops/app/pgdb/README.md]: PHP?

File: traffic_ops/app/db/admin.pl [hunk 2]
```diff
@@ -144,7 +144,7 @@ sub get_dsn_no_dbname {
 sub parse_dbconf_yml_mysql_driver {
```
Comment (dewrich) [inline traffic_ops/app/db/admin.pl]: Should this be renamed to 'parse_dbconf_yml' now?

**Comment by dewrich** [comment]
Which version of Postgres did you use?

File: traffic_ops/app/pgdb/README.md (hunk 1 above)
Comment (knutsel) [inline traffic_ops/app/pgdb/README.md]: best tool i could find. It's a one time use to migrate existing databases.

File: traffic_ops/app/db/admin.pl (hunk 2 above)
Comment (knutsel) [inline traffic_ops/app/db/admin.pl]: No. There should be a new tool created in the pgb dir. db is mysql specific (and default for now)

**Comment by knutsel** [comment]
Using postgres 9.4

**Comment by knutsel** [comment]
Closing this for now; we'll try to add PostgreSQL support in 2.0, so we don't have to get it working w Mojolicious.

**Pull request closed without merging**